homeassistant/components/tuya/siren.py

```diff
@@ -83,6 +83,7 @@ class TuyaSirenEntity(TuyaEntity, SirenEntity):
     """Tuya Siren Entity."""
 
     _attr_supported_features = SirenEntityFeature.TURN_ON | SirenEntityFeature.TURN_OFF
+    _attr_name = None
```

Comment:
This looks good, but the entity descriptions also provide those (and can be removed?)